app/services/github-state.js

```diff
@@ -0,0 +1,73 @@
+import Ember from 'ember';
+
+const { get, Service, inject: { service } } = Ember;
+
+function dec2hex(dec) {
```

Comment:
This is a bit confusing as the `2` could mean digits, although I know this is a conversion function. Maybe a rename.